back/src/utils/concatNames.ts

```diff
@@ -0,0 +1,4 @@
+import { ConventionValidator } from "shared";
+
+export const concatNames = (validator: ConventionValidator): string =>
```

Comment:
```suggestion
export const concatValidatorNames = (validator: ConventionValidator): string =>
```

Et peut être bouger la méthode dans `shared/src/convention/convention.ts` (vu que c'est indissociable de la notion de convention) ?